### ISSUE TITLE
Add Dorothy Matrix M

### ISSUE THREAD

**Issue by drj11** [google/fonts#7744]
**Font Project Git Repo URL:**

https://gitlab.com/drj11/font-dorothy/-/tree/googlefonts?ref_type=heads

**Super short description of the Font Family:**

A dot matrix style font for an authentic 1980s vibe.

**Requirements:**

Right now the upstream repo does not meet the requirements, but i am interested in a vibe check. If Google Fonts wants to add Dorothy Matrix M to its library, then i would be most willing to meet all of the requirements. The big blockers right now are:

- Doesn't use UFO tooling, instead uses esoteric homegrown tooling
- Latin Core isn't completely covered
- isn't licensed under OFL.

But if you say you want the font, i will fix all of these.

There are a couple of other minor issues that my `fontbakery` checks show, but they are fixable too.

Google Fonts will publish only fonts that match its requirements. Please familiarize yourself with the complete documentation in the [Google Fonts Guide](https://googlefonts.github.io/gf-guide/) (GF-Guide) and ensure your font project complies with them before submitting the font family. You can also use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template), which will help you create a repository that follows the needed structure and includes build requirements.

By filling this issue, you can confirm the project meets the  requirements (by ticking the cases or putting x between the square brackets in text mode):

- [x] I read, agree, and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
- [ ] The entire font project is licensed under the [OFL](https://openfontlicense.org/open-font-license-official-text/)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
- [ ] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/blob/main/data/results/txt/nice-names/GF_Latin_Core.txt)
- [x] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [x] I am the author of the font, or the author agreed to this submission
- [x] I will maintain the repository and participate in the onboarding process (addressing, solving, and responding to issues, merging pull requests, etc)

**Image:**
![image](https://github.com/google/fonts/assets/1215412/3b9dcdca-0225-4db4-b6fd-00ef07176051)

**Comment by vv-monsalve** [comment]
HI @drj11, thanks for your submission! We'll review it in our next intake meeting and inform you about its approval.

In the meantime, please remember that Latin Core (linked above) is our required glyphset for new fonts. The font is missing ~100.

<img width="1609" alt="Screen Shot 2024-07-10 at 15 53 31" src="https://github.com/google/fonts/assets/48698976/38bb1da5-b6ca-44bc-888f-b827775690fb">

**Comment by drj11** [comment]
thanks for scheduling it.

I'm aware of the Latin Core requirements, and as discussed in my opening note, i'm quite happy to add those if you agree in principle. Also the UFO thing and the licensing. I already have UFO tooling in a sibling project, so that shouldn't be a problem.

**Comment by drj11** [comment]
3 months have passed now. Have you had an intake meeting? Any notes?

**Comment by drj11** [comment]
yeah, shortly after posting that question i found your submissions queue. It is quite substantial. Good Luck!

**Comment by vv-monsalve** [comment]
Thanks for your understanding!

**Comment by davelab6** [comment]
@drj11 thanks for your patience with us and apologies for the huge delay. This is great work and I'm happy to accept it :) I expect we may get to it in Q3.

**Comment by drj11** [comment]
I am making good progress on adding missing glyphs. I have a few questions about localised glyphs in the `GF_Latin_Core`, which might be easier to ask/answer here, than have me wade through the documentation for glyphsets/shaperglot/fontbakery. Apologies if this is clearly answered elsewhere and i have missed it.

- `idotless` (`i.loclTRK` in AGL-compatible compiled font-file); as i understand it, this is an alias of `i` that is used so that smallcaps `i` correctly gets a dot accent in Turkish. Do you require the glyph in fonts with no smallcaps? Do you require the corresponding `locl` Open Type Layout feature?
- `periodcentered.loclCAT` and `periodcentered.loclCAT.case`; are these required if they are exactly the same as `periodcentered`; do you require corresponding `locl` Open Type Layout features, and what do you expect those features to do?
- `caroncomb.alt` what is this going to do? In my Glyphs files which define this in the normal way (in other fonts), of course it correctly combines to make `dcaron` `lcaron` and `tcaron`; but i don't see why it should be in the exported font-file: there are no Open Type Layout features that use it.

**Comment by drj11** [comment]
i have updated the branch https://gitlab.com/drj11/font-dorothy/-/tree/googlefonts?ref_type=heads (by force push, better reset your tracking branches if you have a clone of the repo).
